GliaWidgets/Theme/Alert/ConfirmationAlertConfiguration.swift

```diff
@@ -13,6 +13,7 @@ public struct ConfirmationAlertConfiguration {
     public var positiveTitle: String?
 
     /// Indicates whether the action button's colors should be switched.
+    @available(*, deprecated, message: "Not used")
```

Comment:
do we have another interface for setting up color for switchButtonBackgroundColors? is it about tint color?

Reply:
this is a one odd property, it was to hack positive action into negative themed one (end engagement confirmation yes is positive in the sense we should take action, but negative in the style, since it's destructive). Not sure if any of the integrators would ever use this. The new alerts handle it differently from the get go, eg. action can have style per whetherr it's a destrucive or normal. Hence this property will render useless. Or shuld we preserve the functionality?